docs/configuration.md

```diff
@@ -66,6 +66,7 @@ report/
 | **DEFAULT_BRANCH**                | `master`              | The name of the repository default branch. Warning: In new github repositories, master branch is named `main`, so you need to override this value with `main`                    |
 | **DEFAULT_WORKSPACE**             | `/tmp/lint`           | The location containing files to lint if you are running locally.                                                                                                                |
 | **DISABLE_ERRORS**                | `false`               | Flag to have the linter complete with exit code 0 even if errors were detected.                                                                                                  |
+| **EXCLUDED_DIRECTORIES**          | \[...\]               | Customize excluded directory list. Default list of directories is stored in `megalinter/utils.py` in `list_excluded_directories` function.                                       |
```

Comment:
You can describe it as "Additional excluded directories" ? It would be more an append to the hardcoded ones than a replacement

Reply:
Maybe I should rename parameter in this case? ADDITIONAL_EXCLUDED_DIRECTORIES ?

Reply:
nop it's ok you can let it like that
But i'd like you to remove the reference to the method... a user should not have to read the sources to know how a parameter works ^^
I'll update build.py to dynamically generate the hardcoded list of excluded directories in the documentation